examples/plugin/README.md

```diff
@@ -0,0 +1,90 @@
+# Plugin Example
+
+This is a project to demo how ORT can be extended with plugins.
+
+## Plugin Architecture
+
+ORT uses the Java [ServiceLoader](https://docs.oracle.com/javase/9/docs/api/java/util/ServiceLoader.html) to load
```

Comment:
May also here use the link https://docs.oracle.com/javase/tutorial/ext/basics/spi.html?